website/project/model.py

```diff
@@ -860,6 +860,7 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     primary_institution = fields.ForeignField('institution')
 
     affiliated_institutions = fields.ForeignField('institution', list=True)
+    visit = fields.IntegerField(default=0)
```

Comment:
It looks like the only way of determining what is or is not a conference submission is by [node's tags](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/conferences/views.py#L225)